src/server/controller/CurvytronController.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Curvytron Controller
+ *
+ * @param {Collection} clients
+ */
+function CurvytronController(clients) {
+    // create socket for all clients
+    this.clientGroup = new SocketGroup(clients);
```

Comment:
Yeah you're right, it wasn't clear. Maybe we should refactor/rename these controllers, there is definitely something ambiguous here.